src/client/pages/main/components/Pics/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import styles from './styles.pcss';
+
+const pics = [
+  {
+    img: 'static/strategy.jpg',
+    alt: 'Strategy',
+    content: (
+      <span className={styles.text}>
+        Strategy
+      </span>
```

Comment:
In this PR I can see 18 eslint errors (maybe because I use windows). For example in this file, we have 7 eslint errors. Please, fix them.